scripts/install/functions.sh

```diff
@@ -39,7 +41,7 @@ function cmd_clean {
     printh "Cleaning old spartacus installation workspace"
 
     delete_dir ${BASE_DIR}
-    delete_dir storage
+    delete_dir ${VERDACCIO_STORAGE_DIR}
```

Comment:
This change was done by the way to improve readability of the code (to know what means the "magic string" `storage`)